pkg/oc/cli/deploymentconfigs/history.go

```diff
@@ -65,7 +67,13 @@ func (h *DeploymentConfigHistoryViewer) ViewHistory(namespace, name string, revi
 		}
 
 		buf := bytes.NewBuffer([]byte{})
-		kinternalprinters.DescribePodTemplate(desired, kinternalprinters.NewPrefixWriter(buf))
+
+		// TODO: remove this once we have external describers
```

Comment:
We updated similar commands upstream to use the `success` message printer for this: https://github.com/kubernetes/kubernetes/pull/65880/files#diff-6685ad8d5708269bc92e3f65a2b1839eR162

This way, we use the normal genericclioptions printer, and avoid using humanreadable bits / converting back to internal